pyanaconda/modules/payload/payloads/payload_base.py

```diff
@@ -123,3 +123,27 @@ def publish_handler(self):
         :rtype: string
         """
         pass
+
+    @abstractmethod
+    def pre_install_with_tasks(self):
+        """Create pre installation tasks.
```

Comment:
Regarding the with_tasks and WithTasks method, for the consistency, it seems to me that we are describing what is the task doing rather than that we are creating the task(s). So something like "Prepare for installation" or "Do pre-installation steps / tasks?".
And for the install_with_tasks below: "Install the payload" instead of "Create tasks for installation."

Reply:
Fixed. Thanks for pointing that out.